PowerEditor/src/NppCommands.cpp

```diff
@@ -708,7 +708,7 @@ void Notepad_plus::command(int id)
 			}
 			else if (id == IDM_EDIT_SORTLINES_INTEGER_DESCENDING || id == IDM_EDIT_SORTLINES_INTEGER_ASCENDING)
 			{
-				pSorter = std::unique_ptr<ISorter>(new NaturalSorter(isDescending, fromColumn, toColumn));

```

Comment:
NaturalSorter class is not necessary anymore - it should be removed from the code.

Reply:
Should I delete NaturalSorter from the code entirely? ie, delete its implementation from sorter.h? Or should I just delete its usage in NppCommands.cpp?

Reply:
NaturalSorter should be removed from the code entirely, since it is not used anymore.

Reply:
NaturalSorter is now removed from Sorter.h.